Dependencies/xr/Source/Shared/XRHelpers.h

```diff
@@ -0,0 +1,41 @@
+#pragma once
+#include <XR.h>
+
+namespace xr
+{
+	/**
```

Comment:
This file has all tabs instead of spaces.

Reply:
VS code why :(